packages/lesswrong/components/users/DialogueMatchingPage.tsx

```diff
@@ -16,8 +16,21 @@ import ReactConfetti from 'react-confetti';
 import { Link } from '../../lib/reactRouterWrapper';
```

Comment:
First impression: this file is much too big. Github gives me a "need to manually click to load". Components in here should be factored out into their own files.

Reply:
This is non blocking and there seems to be some philosophical disagreement about whether big files are a problem; resolving for now and maybe a future PR will address